build.sbt

```diff
@@ -385,7 +385,7 @@ lazy val perf = project
 lazy val `cromwell-drs-localizer` = project
   .withExecutableSettings("cromwell-drs-localizer", drsLocalizerDependencies, drsLocalizerSettings)
   .dependsOn(`cloud-nio-impl-drs`)
-  .dependsOn(common % "test->test")
+  .dependsOn(common)
```

Comment:
Any idea what this change is about? Seems unrelated to the rest of this PR.

Reply:
I had no idea what this was but found this SO thread: https://stackoverflow.com/questions/8193904/sbt-test-dependencies-in-multiprojects-make-the-test-code-available-to-dependen

Looks like this change removes a dependency on the test code in `common`, I assume because it wasn't actually being used.